src/scaraplate/automation/git.py

```diff
@@ -0,0 +1,298 @@
+import contextlib
+import datetime
+import logging
+import tempfile
+import uuid
+from pathlib import Path
+from typing import Dict, Iterator, List, Optional
+from urllib.parse import urlparse, urlunparse
+
+from pkg_resources import get_distribution
```

Comment:
`pkg_resources` comes with setuptools, which should be present when installing packages with `pip`.

For os-specific packages this indeed might be an issue (for example, Debian seems to ship it separately from setuptools: https://packages.debian.org/stable/python3-pkg-resources), but we don't provide os-specific packages, so these dependencies should be handled downstream.

Reply:
That's all right *if* packages being installed via pip. System packages are not of that kind and there you cannot relay on pip presence and package maintainers will be very surprised about such kind of things. That was my point.

Reply:
I see. Any idea how to make this easier to spot? I assume we cannot just add explicit dependency on `pkg_resources` in `install_requires` given that this package comes from `setuptools`. Perhaps a requirement on `setuptools` would suffice? The downstream maintainers should know if they split `setuptools` or not.

Reply:
I have the only solution is to push setuptools to install_requires. Or remove the scaraplate version from the base template. It's quite awkward that stdlib itself cannot figure out which versions of such packages are in site-packages, but that's what we have in Python.

Reply:
> I have the only solution is to push setuptools to install_requires.

Ok, I'll try that.

> Or remove the scaraplate version from the base template.

But I like it there! Different scaraplate versions might produce different results, so it is good to have that information there.

> It's quite awkward that stdlib itself cannot figure out which versions of such packages are in site-packages, but that's what we have in Python.

Well, we could've had it, but the proposal was rejected 12 years ago: https://www.python.org/dev/peps/pep-0365/

Reply:
> Well, we could've had it, but the proposal was rejected 12 years ago: https://www.python.org/dev/peps/pep-0365/

What a shame! Rejecting tools, but pushing forward metadata format (from there those tools get to know about what things are around). Hope this will get revisited one day.

Reply:
Me too.

Just pushed `setuptools`. If Travis build passes -- I'm merging this. 🚀

Reply:
Yeah, :ship:  it!